src/components/Missions.js

```diff
@@ -15,6 +15,12 @@ const Mission = () => {
     dispatch(reserveMission(missionId));
   };
 
+  const leaveMission = (e) => {
```

Comment:
- [ ]  When the UI render is throwing the following warnings, In my opinion, it would be nice if you avoid them, what do you think? 
![Captura de Pantalla 2021-12-15 a la(s) 8 54 22](https://user-images.githubusercontent.com/88604987/146210778-4654bda9-4fef-41b0-a049-6d0bc9ee4917.png)